src/transpiling/mod.rs

```diff
@@ -238,7 +238,7 @@ impl ParsedSource {
       }
       let mut src = String::from_utf8(buf)?;
       let mut map: Option<String> = None;
-      {
+      if options.source_map || options.inline_source_map {
```

Comment:
Opened https://github.com/denoland/deno_ast/issues/217 for the future (breaking change)